rabbitmq/files/entrypoint.sh

```diff
@@ -1,6 +1,29 @@
-{%- from "rabbitmq/map.jinja" import server with context -%}
+{%- from "rabbitmq/map.jinja" import server,cluster with context -%}
 #!/bin/bash -e
 
+{%- if cluster.enabled %}
+
+# if NUM_NODES not passed, default to 3
+if [ -z "$NUM_NODES" ]; then
+  NUM_NODES=3
+fi
+
+# loop through number of nodes
+for NUM in `seq 1 $NUM_NODES`; do
+  NODE_SERVICE_HOST="RABBITMQ_SERVER_NODE0${NUM}_SERVICE_HOST"
+
+  # if set
+  if [ -n "${!NODE_SERVICE_HOST}" ]; then
+    # if not its own IP, then add it
+    if [ $(expr "$HOSTNAME" : "rabbitmq-server-node0${NUM}") -eq 0 ]; then
```

Comment:
I wouldn't force user to match rabbitmq-server-node pattern of hostname.
Try this pattern:
```
if [[ "$HOSTNAME" =~ ^.+0?${NUM}$ ]]
```